iceoryx2-bb/log/src/logger/console.rs

```diff
@@ -170,7 +171,7 @@ impl crate::Log for Logger {
                     Self::print_origin(log_level, &origin_str);
                 }
                 true => std::eprint!(
-                    "{}{:9} {} ",
+                    "{}{} {} ",
```

Comment:
Removed this indentation so that logs with and without `origin` are aligned.

Reply:
Hmm, what do you think about moving the origin at the second position, after the log message? Maybe not in this PR but just in general.

Reply:
@elBoberido @elfenpiff I will leave it to you guys to decide. Could you create an issue if it's decided that we want to make the change.